app/__init__.py

```diff
@@ -8,10 +9,77 @@
 
 
 def create_app(config_name):
+    from app.models import BucketList
+
     app = FlaskAPI(__name__, instance_relative_config=True)
     app.config.from_object(app_config[config_name])
     app.config.from_pyfile('config.py')
     app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
     db.init_app(app)
 
+    @app.route('/bucketlists/', methods=['POST', 'GET'])
+    def bucketlists():
+        if request.method == 'POST':
+            name = str(request.data.get('name', ''))
+            if name:
+                bucketlist = BucketList(name=name)
+                bucketlist.save()
+                response = jsonify({
+                    'id': bucketlist.id,
+                    'name': bucketlist.name,
+                    'date_created': bucketlist.date_created,
+                    'date_modified': bucketlist.date_modified
+                })
+                response.status_code = 201
+                # import pdb
+                # pdb.set_trace()
+                return response
+        else:
+            bucketlists = BucketList.get_all()
+            all_bucketlists = []
+            for bucketlist in bucketlists:
+                obj = {
```

Comment:
If we find ourselves repeating this pattern, we might want to abstract it into a function.
```python
obj = {
    'id': bucketlist.id,
    'name': bucketlist.name,
    'date_created': bucketlist.date_created,
    'date_modified': bucketlist.date_modified
}
```